test/xavierTests.dpr

```diff
@@ -0,0 +1,26 @@
+program xavierTests;
```

Comment:
@nunopicado please rename. Use same pattern that we use in James: `xavier.tests.gui.delphi`.